README.rst

```diff
@@ -26,9 +26,12 @@ versions of the `LLVM-C API`_ should be basically effortless.
 
 To try it out install LLVM and get `llvmcpy` using `pip`::
 
-    sudo apt-get install llvm
+    sudo apt-get install llvm libllvm3.8
```

Comment:
There's no `libllvm` package, only specific-to-a-release ones.  I drafted other versions that explained this but it seemed too wordy.  But `libllvm` is necessary because it installs the `.so`s and `llvm` does not.